default.py

```diff
@@ -490,38 +511,22 @@ def onClick(self, controlId):  # pylint: disable=invalid-name
                 elif controlId == 230:  # episode is clicked
                     self.init('game/episode')
                     video_id = self.games_list.getSelectedItem().getProperty('id')
-                    video_streams = gp.get_stream(video_id, 'video', username=username)
-                    if video_streams:
-                        addon_log('Video-Streams: %s' % video_streams)
-                        bitrate = self.select_bitrate(video_streams.keys())
-                        if bitrate:
-                            video_url = video_streams[bitrate]
-                            self.play_url(video_url)
+                    episode_stream_url = self.select_stream_url(gp.get_stream(video_id, 'video', username=username))
+                    if episode_stream_url:
+                        self.play_url(episode_stream_url)
                     else:
                         dialog = xbmcgui.Dialog()
                         dialog.ok(language(30043), language(30045))
                 elif controlId == 240:  # Live content (though not games)
                     show_name = self.live_list.getSelectedItem().getLabel()
                     if show_name == 'NFL RedZone - Live':
-                        rz_live_streams = gp.get_stream('redzone', username=username)
-                        if rz_live_streams:
-                            bitrate = self.select_bitrate(rz_live_streams.keys())
-                            if bitrate:
-                                rz_live_url = rz_live_streams[bitrate]
-                                self.play_url(rz_live_url)
-                        else:
-                            dialog = xbmcgui.Dialog()
-                            dialog.ok(language(30043), language(30045))
+                        rz_stream_url = self.select_stream_url(gp.get_stream('redzone', username=username))
+                        if rz_stream_url:
+                            self.play_url(rz_stream_url)
```

Comment:
The reason I removed it is because select_stream_url will return None if the bitrate dialog is canceled, so that's not really anything that fails. I'll restore it and explicitly check for 'if live_streams is not False'.